cli/args/flags.rs

```diff
@@ -26,6 +26,13 @@ use crate::util::fs::canonicalize_path;
 
 use super::flags_net;
 
+/// Validate that the url have a valid file scheme
+/// This check needs to be made before `to_file_path`
+fn have_valid_file_scheme(url: &Url) -> bool {
+  // url `to_file_path` does not validate scheme
+  url.as_str().starts_with("file:///")
```

Comment:
Why not `url.scheme() == "file"`?

Reply:
file:/ and file:// are both invalid but still have scheme==file